app/views/dashboards/subscriptions.html.erb

```diff
@@ -0,0 +1,39 @@
+<% title "Subscriptions #{@source.title}" %>
```

Comment:
This won't quite work if/when we extend subscriptions to other objects, like comments, in the future (`Comment` doesn't have a `title`). We can cross that bridge when we get there. It's a quick update. We also don't have the concept of a "comment dashboard" so it's possible that view would live somewhere else anyways.